src/components/CategoryOverview/CategoryOverview.scss

```diff
@@ -132,6 +133,14 @@
   }
 }
 
+.indicatorCards {
+  justify-content: center;
+
+  @include for-tablet-landscape-up {
+    flex-grow: 2;
```

Comment:
The typical way to say "fill the available space" is just `flex: auto`. See these [commonly desired effects](https://www.w3.org/TR/css-flexbox-1/#flex-common) in the spec if you're interested.

`flex: 2` would only be required if you wanted to balance the space taken with other elements with a 2:1 ratio or something.